custom/conf/app.example.ini

```diff
@@ -2555,7 +2555,8 @@ LEVEL = Info
 ;LIMIT_SIZE_SWIFT = -1
 ;; Maximum size of a Vagrant upload (`-1` means no limits, format `1000`, `1 MB`, `1 GiB`)
 ;LIMIT_SIZE_VAGRANT = -1
-
+;; Resign RPM packages (using v4 format, not compatible with CentOS 7 or older)
+;RPM_SIGN_ENABLED  = false
```

Comment:
Is this option necessary?

Reply:
> Is this option necessary?

@lunny It may be necessary, or perhaps some people's RPMs have already been signed and do not need to be re-signed by Gitea.

Reply:
Maybe a post parameter could be given when uploading to indicate sign or not sign?

Reply:
I agree that this should not be a global option but in case such a org option most probably

Reply:
I think we can have it as a global option initially, as setting a default to false doesn't change the behaviour, then a follow up PR could be done to change it to per user/org option, as that would involve template changes, etc., and this way we can keep this PR smaller.

Reply:
> I think we can have it as a global option initially, as setting a default to false doesn't change the behaviour, then a follow up PR could be done to change it to per user/org option, as that would involve template changes, etc., and this way we can keep this PR smaller.

@techknowlogick I added a request parameter that allows controlling whether to sign during the upload, as well as configuring whether to enable signing by default.